docs/gathered-data.md

```diff
@@ -159,6 +161,8 @@ collects all the ClusterOperators definitions and their resources.
 The Kubernetes api https://github.com/openshift/client-go/blob/master/config/clientset/versioned/typed/config/v1/clusteroperator.go#L62
 Response see https://docs.openshift.com/container-platform/4.3/rest_api/index.html#clusteroperatorlist-v1config-openshift-io
 
+* Previous location of operators related resources in the older archives: config/clusteroperator/{kind}-{name}
```

Comment:
I am not sure if this was ever released, but it's probably true that we got some archives (with this structure) from CI/testing clusters.